android/samples/sample-gltf-viewer/build.gradle

```diff
@@ -30,7 +30,7 @@ android {
     compileSdkVersion versions.compileSdk
     defaultConfig {
         applicationId "com.google.android.filament.gltf"
-        minSdkVersion 19
+        minSdkVersion 21
```

Comment:
we can probably replace this with `versions.minSdk` like other samples: https://github.com/google/filament/blob/main/android/samples/sample-material-builder/build.gradle#L31C1-L32C1

Reply:
Good point. Done.